fdbserver/Status.actor.cpp

```diff
@@ -2358,6 +2358,40 @@ ACTOR static Future<JsonBuilderObject> clusterSummaryStatisticsFetcher(
 	return statusObj;
 }
 
+ACTOR static Future<JsonBuilderObject> blobWorkerStatusFetcher(
```

Comment:
This will work, but it would probably be even better to request this information from the blob manager since that is one process instead of many? Kind of like how status talks to Data Distribution instead of all of the Storage Servers?
The blob manager doesn't have this information in its current metrics but we could easily add it, as it knows how many ranges it has assigned to each worker.

Reply:
Update based on offline conversation: there are other useful metrics to pull from each worker, similar to storage servers, that will be added in a later PR, so adding the framework to pull from blob workers makes sense